cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,153 @@ Cask Data Application Platform Release Notes
+
+- :cask-issue:`CDAP-13894` - Fixed a bug that metrics query by time series will be incorrect after a certain amount of time.
+
+- :cask-issue:`CDAP-13959` - Fixed a bug where profile metrics is incorrect if an app is deleted.
+
+- :cask-issue:`CDAP-13965` - Fixed a deprovisioning bug when cluster creation would fail.
+
+- :cask-issue:`CDAP-13988` - Fixed an error that where TMS publishing was retried indefinitely, if the first attempt failed.
+
+- :cask-issue:`CDAP-14040` - Added format and suppress query params to the program logs endpoint to match the program run logs endpoint
+
+- :cask-issue:`CDAP-14076` - Fixed a race condition in MapReduce that can cause a deadlock.
+
+- :cask-issue:`CDAP-14098` - Fixed a resource leak in preview feature.
+
+- :cask-issue:`CDAP-14107` - Fixed a bug that would cause RDD versions of the dynamic scala spark plugins to fail
+
+- :cask-issue:`CDAP-14132` - Made all CDAP examples to be compatible with Spark 2.
+
+- :cask-issue:`CDAP-14154` - Fixed a bug where profiles were getting applied to all program types instead of only workflows
+
+- :cask-issue:`CDAP-14185` - Added Google Cloud PubSub Realtime Source
+
+- :cask-issue:`CDAP-14203` - Fixed a race condition by ensuring that a program is started before starting runtime monitoring for it.
+
+- :cask-issue:`CDAP-14211` - Bugfix: Fixes runs count for pipelines in UI to show correct number instead of limiting to 100.
+
+- :cask-issue:`CDAP-14220` - Added worker and master disk size properties to the Dataproc provisioner
+
+- :cask-issue:`CDAP-14223` - Fixed an issue, where Dataproc client was not being closed, resulting in verbose error logs.
+
+- :cask-issue:`CDAP-14261` - Fixed a bug that could cause the provisioning state of stopped program runs to be corrupted.
+
+- :cask-issue:`CDAP-14271` - Fixed a bug that caused data prep to be unable to list buckets in a Google Cloud Storage connection in certain environments
+
+- :cask-issue:`CDAP-14298` - Improves operational behavior of the dataset service.
+
+- :cask-issue:`CDAP-14303` - Fixed a bug that Dataproc provisioner is not able to provision a singlenode cluster.
+
+- :cask-issue:`CDAP-14321` - Added csv, tsv, delimited, json, and blob as formats to the File source
+
+- :cask-issue:`CDAP-14372` - Fixed dataprep transform to make directives optional. If none are given, the transform is a no-op.
+
+- :cask-issue:`CDAP-14390` - Fixed a bug where dataprep could not read json or xml files on Google Cloud Storage.
+
+- :cask-issue:`CDAP-14395` - Fixed the dataproc provisioner to use full API access scopes so that Spanner and PubSub are accessible by default.
+
+- :cask-issue:`CDAP-14397` - Files without extension will be treated as text files.
+
+- :cask-issue:`CDAP-14398` - Limiting max number of files showed in the S3 and GCS file browser to 1000.
+
+- :cask-issue:`CDAP-14432` - Bugfix: Fixes data preparation, file upload feature to properly create workspace.
```

Comment:
Fixed...

Reply:
Actually, unless this was broken in 5.0, we should just remove this. This may have been a regression that got fixed within the release.